dom-events.tar/dom-events/buttons.js

```diff
@@ -0,0 +1,24 @@
+function handleClick(event){
+  console.log('button clicked');
+  console.log(event);
+  console.log(event.target);
+  document.querySelector('.click-button')
+  if (!$clickButton) throw new Error('The $clickBtn query failed');
+  element.addEventListener('click',handleClick());
```

Comment:
lines 5-7 should exist outside of the function definition!

line 5 should have the result assigned to a variable, in your case named `element` to match what you have in line 7. So something like this: `const element = document.querySelector('.click-button');`

Also, the `handleClick` argument on line 7 should NOT have the `()` surrounding them or else the callback function won't work properly! I'll talk about this in Q&A. It should look like this:
`element.addEventListener('click', handleClick));`